spec/features/favorites/index_spec.rb

```diff
@@ -46,6 +46,21 @@
         expect(page).to have_link(@pet_2.name)
         expect(page).to have_css("img[src='#{@pet_2.image}']")
       end
+
+    it "Sees no favorites message on index page user has no favorites" do
+      visit "/favorites"
+
+      expect(page).to have_content("You haven't favorited any pets yet.")
     end
 
+    it "Sees no no-favorites message on index page if user has favorites" do
```

Comment:
Nice job making a second test to make sure the no favorites message doesn't show when we have favorites.